Core/clim-core/presentations/translators.lisp

```diff
@@ -457,17 +457,11 @@ and used to ensure that presentation-translators-caches are up to date.")
         (loop for context in input-context
               do (mopscp context presentation)))))
 
-(defun window-modifier-state (window)
-  "Provides default modifier state for presentation translator functions."
-  ;; There are many things which may go wrong (NULL window, sheet
-  ;; without a port etc). We clamp errors to 0 meaning "no modifiers".
-  (or (ignore-errors (pointer-modifier-state (port-pointer (port window)))) 0))
```

Comment:
I understand that it was always factually 0, but should it be? shouldn't we update the pointer modifier state?

Reply:
or use port-modifier-state. I imagine that this is used for the pointer documentation, so when control is pressed I'd expect that the modifier state is set up accordingly. It seems from the next commit, that pointer-modifer-state may change by means of the method pointer-update-state.

Reply:
I was in doubt to use `port-modifier-state` or remove this function, I opt for the second because now the pointer documentation works good despite this function return always 0 so I deduce that the pointer documentation machinery doesn't need this function.

Reply:
currently we synthesize the motion event for the pointer documentation when necessary.